src/istio/chart/templates/tls-validation-job.yaml

```diff
@@ -0,0 +1,95 @@
+# Copyright 2024 Defense Unicorns
+# SPDX-License-Identifier: AGPL-3.0-or-later OR LicenseRef-Defense-Unicorns-Commercial
+
```

Comment:
Can we make this all conditional on `not tls.enabled`? I think the check is only applicable in that case, so we might as well remove the complexity if TLS is enabled, rather than having that check in the job.